### DISCUSSION TITLE
Version 0.20.2 Warning

### DISCUSSION THREAD

**Discussion by RoyKO1405** [FreeCAD/FreeCAD#9907] [Code contribution discussions]
I have just down loaded the latest version and going through the scripts at the bottom of the page I find this 11:29:02  You seem to be using FreeCAD version 0.20 from 2022 which is quite old. Some functionality of latest versions might be missing. What does it mean please?

**Comment by luzpaz** [comment]
This is code contributing discussion space. Not for general help.  

What it probably means is a hint to use the latest development version of FreeCAD (at the time of this writing is v0.21.x and can be downloaded from https://github.com/FreeCAD/FreeCAD-Bundle/releases/tag/weekly-builds). The FreeCAD software release process is slated to improve in the future but for now there is a big lag between the stable version and development version. What ends up happening as development goes on is less and less fixes and features are backported to the stable version. And so a gap starts to form and widen, which by the end of the release cycle is quite large (1000s of code commits ahead of stable). 

This again, is hopefully going to improve in the future. For now, the unofficial recommendations is following the development branch and downloading it on a weekly or daily basis.
